components/climber.py

```diff
@@ -1,13 +1,45 @@
+import wpilib
+from rev import CANSparkMax
+from ids import SparkMaxIds
+
+
 class ClimberComponent:
     def __init__(self) -> None:
         self.deployed = False
 
     def deploy(self) -> None:
         self.deployed = True
 
+    def execute(self) -> None:
+        if self.deployed:
+
+            def __init__(self) -> None:
+                self.deployed = False
+
+    def deploy(self) -> None:
+        self.deployed = True
+
     def execute(self) -> None:
         if self.deployed:
             # Deploy the climber
-            pass
+            self.climbingmotor = CANSparkMax(
+                SparkMaxIds.climber, CANSparkMax.MotorType.kBrushless
+            )  # using a Neo with 4:1 4:1 3:1 ratio
+            self.limitswitch = wpilib.LimitSwitch(2)
+
+            def motorCurrent():
+                return CANSparkMax.getOutputCurrent
+
+            TriggeringLevels = 100  # TriggeringLevels Placeholder for the current levels needed to stop
+            limitswitchHit = False  # placeholder for limit switch variable
+            if limitswitchHit == False:
+                self.drive.motor(0.5, 0)
+            elif limitswitchHit == True & motorCurrent() <= TriggeringLevels:
```

Comment:
Since this is the else block of an if statement checking if the limit switch was not hit, the first half of this check will always pass making it unnecessary